src/main/java/ecostruxure/rate/calculator/bll/service/TeamService.java

```diff
@@ -57,22 +57,12 @@ public Team update(UUID teamId,Team team) throws Exception {
     }
 
     /**
-     * Create a new team.
-     *
-     * @param team the team to create. Must not be null.
-     * @return the created team that has same data as the input team, but with ID set.
-     * @throws Exception                if an error occurred in creating the team.
-     * @throws NullPointerException     if the input team or team name is null.
-     * @throws IllegalArgumentException if the team name is empty.
+     * This one is used by controller to create profiles
```

Comment:
vær mere konkret med hvad metoden gør istedet for det du har skrevet kunne det bare være "Creates a new team."